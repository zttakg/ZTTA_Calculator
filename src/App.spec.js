import { shallowMount, createLocalVue } from "@vue/test-utils";
import Vuex from "vuex";

import App from "./App";

const localVue = createLocalVue();
localVue.use(Vuex);

describe("<App /> component", () => {
  it("should render correct without props", () => {
    const store = new Vuex.Store({
      state: {}
    });

    const wrapper = shallowMount(App, {
      store,
      localVue,
      stubs: {
        "router-view": true
      }
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
