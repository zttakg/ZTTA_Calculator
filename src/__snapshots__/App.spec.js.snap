// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> component should render correct without props 1`] = `
<div id="app">
  <!---->
</div>
`;
